src/mongo_logger.ts

```diff
@@ -760,15 +761,27 @@ export class MongoLogger {
     this.maxDocumentLength = options.maxDocumentLength;
     this.logDestination = options.logDestination;
     this.logDestinationIsStdErr = options.logDestinationIsStdErr;
+    this.willLog = this.createWillLog();
```

Comment:
```suggestion
    this.severities = this.createLoggingSeverities();
```
Suggest renaming the structure and method to create it.